cheroot/ssl/pyopenssl.py

```diff
@@ -150,14 +150,26 @@ def readline(self, size=-1):
             size,
         )
 
-    def sendall(self, *args, **kwargs):
-        """Send whole message to the socket."""
+    def read(self, *args, **kwargs):
+        """Read from the wrapped socket, with retry."""
         return self._safe_call(
-            False,
-            super(SSLFileobjectMixin, self).sendall,
+            True,
+            super(SSLFileobjectMixin, self).read,
             *args, **kwargs
         )
 
+    def sendall(self, *args, **kwargs):
+        """Send whole message to the socket.
+
+        Not supported due to https://github.com/pyca/pyopenssl/issues/176.
+        Until that bug is fixed, sendall() may throw SSL.WantWriteError, but
+        there is no correct way to retry the call because we don't know how
+        many bytes were already transmitted. We could work around this by
+        reimplementing sendall() using send(), but we don't actually use
+        sendall() anywhere.
```

Comment:
When you say "we don't use" do you mean not even some internal calls hit it? How do we get the error, then?

Reply:
We don't, as far as I know, it's just something I noticed while googling the WantWriteError issue.
There is no occurrence of `sendall` in the whole cheroot project, other than in `pyopenssl.py`. (Also, my server seems to work with this change.)

Reply:
Well, if it's unrelated to the PR, it should be in a separate PR.